README.md

```diff
@@ -1,19 +1,90 @@
+
 
 ## What are Workflows?
 
 Workflows are an easier way to execute and share commands within Warp. They are searchable by name, description, or command and are easily parameterized. See our documentation for more details: [https://docs.warp.dev/features/workflows](https://docs.warp.dev/features/workflows)
 
-## How Do I Access Workflows?
+## How Do I Access Workflows within Warp?
 
-Workflows can be accessed within Warp directly within the app, either through the Command Palette or by pressing `ctrl-shift-r`.
+Workflows can be accessed directly within Warp, either through the Command Palette or by pressing `ctrl-shift-r`.
 
 All public workflows (i.e. workflows within this repo) are also available at [commands.dev](https://www.commands.dev/).
 
 ## Contributing
-
 Contributions are always welcome! If you have a workflow that would be useful to many Warp users, feel free to send a PR to add a Workflow spec.
 
-The existing preset files should serve as an example for the format, while the a more comprehensive description of the file format is available in [FORMAT.md](FORMAT.md). Please request a review from someone on the Warp team once a PR is ready for review.
+All workflows are defined as YAML files within the [`specs/`](specs/) directory. 
+
+### File Format
+A comprehensive description of the file format is available in [FORMAT.md](FORMAT.md).
+Additionally, see the workflow below as an example to quickly get started:
+
+```yaml
+---
+# The name of the workflow.
+name: Uninstall a Homebrew package and all of its dependencies
+# The corresponding command for the workflow. Any arguments should be surrounded with two curly braces. E.g `command {{arg}}`.
+command: "brew tap beeftornado/rmtree\nbrew rmtree {{package_name}}"
+# Any tags that workflow should be categorized with.
+tags:
+  - homebrew
+# A description the workflow.
+description: Uses the external command rmtree to remove a Homebrew package and all of its dependencies
+# List of arguments within the command.
+arguments:
+    # Name of the argument within the command. This must match exactly with the name
+    # of the argument within the command (without the curly braces).
+  - name: package_name
+    # The description of the argument.
+    description: The name of the package that should be removed
+    # The default value for the argument.
+    default_value: ~
+# The source URL for where the workflow was generated from, if any.
+source_url: "https://stackoverflow.com/questions/7323261/uninstall-remove-a-homebrew-package-including-all-its-dependencies"
+# The author of the workflow.
+author: Ory Band
+# The URL of original author of the Workflow. For example, if this workflow was generated from StackOverflow, the `author_url` would be the StackOverflow author's profile page.
+author_url: "https://stackoverflow.com/users/207894"
+# The valid shells where this workflow should be active. If valid for all shells, this can be left empty.
+shells: []
```

Comment:
Forward-thinking: Is this due to some shells (i.e.: fish) having different syntax?  If so, would we want to consider supporting multiple `command` values so that there could be per-shell overrides?

Reply:
yeah that's another fair way of approaching it--I think both are valid fwiw. The reason I prefer writing it like this is that arguments could also differ between the two commands, in which case you end up essentially creating two workflows anyway. It also makes things like author accreditation hard.